js/app.js

```diff
@@ -0,0 +1,71 @@
+
+
+// SECTION 1 ANSWERS:
+// Interpolation is adding a variable in your string concatenation
+// Concatenation is purely combining strings together
+// DRY stands for Don't Repeat Yourself. 
+// It means to condense your code into smaller parts instead of having many repetitive lines. It makes the code neater and more concise.
+// For loops and while loops are ways we've learned so far of implementing DRY Code
+
+// SECTION 2 ANSWERS:
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a < b < c);
+console.log(a === a != d); //still reading false
```

Comment:
this would work:
```
a <= a <= d
```

Reply:
...but not for the reasons we might hope.  i think what they were looking for is:
```
a * a === d